can-define.js

```diff
@@ -288,6 +288,7 @@ make = {
 		events: function(prop, getCurrent, setData, eventType) {
 			return function(newVal) {
 				if (this.__inSetup) {
+					Observation.ignore(getCurrent).call(this);
```

Comment:
Honestly, all three of the changes in #163 were directions from @justinbmeyer and I didn't fully understand them. That was literally my first day.